activities/Blockrain.activity/lib/sugar-web/activity/activity.js

```diff
@@ -5,8 +5,10 @@ define(["webL10n",
         "sugar-web/datastore",
 		"sugar-web/presence",
         "sugar-web/graphics/icon",
-        "sugar-web/graphics/activitypalette"], function (
-    l10n, shortcut, bus, env, datastore, presence, icon, activitypalette) {
+        "sugar-web/graphics/activitypalette",
```

Comment:
You're not supposed to change this file, it's a part of the sugar-web library.

Reply:
Fixed icon at end button and 

> You're not supposed to change this file, it's a part of the sugar-web library.

Fixed